opendm/types.py

```diff
@@ -334,9 +451,8 @@ def __init__(self, root_path, images_path):
         self.odm_georeferencing_latlon = io.join_paths(
             self.odm_georeferencing, 'latlon.txt')
         self.odm_georeferencing_coords = io.join_paths(
-            self.odm_georeferencing, 'coords.txt')
-        self.odm_georeferencing_gcp = io.join_paths(
-            self.odm_georeferencing, 'gcp_list.txt')
+            self.root_path, 'coords.txt') # Todo put this somewhere better
+        self.odm_georeferencing_gcp = io.find('gcp_list.txt', self.root_path)
```

Comment:
This should probably be set to `args.gcp` (as when passing `--gcp <path>` to `run.py`). The GCP file is not always named `gcp_list.txt`.

Reply:
Or at least I didn't think so; is the `--gcp` parameter even used across the solution?

Reply:
That looks like something I hard-coded and didn't fix. Nice catch!